website/thaliawebsite/settings.py

```diff
@@ -651,6 +651,16 @@ def from_env(
     "compressor.finders.CompressorFinder",
 )
 
+NORMAL_STATICFILES_STORAGE = "django.contrib.staticfiles.storage.StaticFilesStorage"
+MANIFEST_STATICFILES_STORAGE = (
+    "django.contrib.staticfiles.storage.ManifestStaticFilesStorage"
+)
```

Comment:
This could be shorter we leave out the testing option, since that is the same value as development in `setting`

Reply:
Bit nitpicky tho :p